krux_marathon_api/marathonapi.py

```diff
@@ -107,86 +96,70 @@ def assign_config_data(self, config_file_data, marathon_app_result):
         marathon_app_result.user                     = config_file_data["user"]                 #updates confirmed
 
     def list_marathon_apps(self, marathon_server):
+        """
+        This method connects to the marathon server and requests all the
+        Marathon Apps.
+        """
         self.logger.info("Listing apps running on marathon")
         current_marathon_apps = marathon_server.list_apps()
-        self.logger.info(current_marathon_apps)
-        print(current_marathon_apps)
-
-    def get_marathon_app(self, marathon_server, config_file_data):
-        self.logger.info("Getting app info from the marthon server")
-        if self.marathon_config_file:
+        return current_marathon_apps
+
+    def get_marathon_app(self, marathon_server, config_file_data, app_id):
+        """
+        Connects to the Marathon Server using the supplied App ID.
+        If we passed the config file and the App isn't found on the server, we
+        try to create it.
+        If the App name was passed on the command line instead, we will just
+        return with an error if the App wasn't found.
+        Input is marathon connection, config file data, and App id.
+        Returns the Marathon App results.
+        """
+        self.logger.info("Getting app info from the marathon server")
+        if config_file_data:
             try:
-                marathon_app_result = marathon_server.get_app(config_file_data["id"])
+                marathon_app_result = marathon_server.get_app(app_id)
             except Exception as e:
```

Comment:
Catch-'em-all error handling like this is generally a bad idea. If you want to handle the case where `app_id` is invalid, isn't there a specific error type for that?